loleaflet/src/layer/tile/CanvasSectionContainer.ts

```diff
@@ -265,6 +265,37 @@ class CanvasSectionContainer {
 		return this.clearColor;
 	}
 
+	/**
+	 * public domain polyfill needed for ie11
+	 * https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Array/includes
+	*/
+	private arrayIncludes<T> (array: Array<T>, searchElement: T /*, fromIndex*/) {
+		'use strict';
+		var O = Object(array);
+		var len = parseInt(O.length) || 0;
+		if (len === 0) {
+			return false;
+		}
+		var n = parseInt(arguments[1]) || 0;
+		var k;
+		if (n >= 0) {
+			k = n;
+		} else {
+			k = len + n;
+			if (k < 0) {k = 0;}
+		}
```

Comment:
Hello, so IE doesn't support "includes" :) Thank you for the fix.

I think there is something wrong here. We can set n = k = 0 directly. You have 1 more variable in parameters. arguments[1] is not the "fromIndex" variable. We may not need the variable "n".

Also, TypeScript has type "array". We don't need to cast it to object.

We can also write something like:

var len: number = array.length; // without "|| 0" part.
var k: number = 0;

With your code, behaviour will seem fine. Because search objects are not numbers. If we would search a number in an array, there would be bugs.

I also read the mozilla page. Thank you for the link.

If you want to add "fromIndex" functionality, we need one more variable. I think it's okay without it. These are all your's to decide:)

Thanks :)

Reply:
@gokaysatir thanks for review. Now I found out that we can replace the `includes` with just `indexOf` :) So no need for implementing on my own.

Reply:
Okay, thank you for the fix :)